src/main/java/com/minecolonies/coremod/util/BlockUtils.java

```diff
@@ -86,6 +87,27 @@ public static boolean shouldNeverBeMessedWith(final Block block)
     }
 
     /**
+     * Gets a rotation from a block facing.
+     *
+     * @param facing the block facing.
+     * @return the int rotation.
+     */
+    public static int getRotationFromFacing(final EnumFacing facing)
+    {
+        switch (facing)
+        {
+            case SOUTH:
+                return 2;
+            case EAST:
+                return 1;
+            case WEST:
+                return 3;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 3 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)